shared/oae/api/oae.api.util.js

```diff
@@ -1656,12 +1656,15 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
                 return '';
             } else {
 
+                input = input.replace(/\*/g, "\\*");
+
                 // Convert the Markdown input string to HTML using marked.js. `gfm`
                 // automatically recognizes text beginning with http: or https: as a URL
                 // and converts it to a link. We also specify that the input should be sanitized.
                 // @see https://github.com/chjj/marked
                 input = markdown(input.toString(), {
                     'gfm': true,
+                    'tables': true,
```

Comment:
The only skinnable element should be the text colour using "Plain text color" value, the lines and BG striping seems a bit overkill to ask administrators to configure, or assigning values to arbitrary styles could be confusing and problematic.